classes/patterns/whole-page-large.html

```diff
@@ -0,0 +1,263 @@
+<!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"60px"}},"className":"page-section-header h2"} -->
```

Comment:
This is just the other pattern 3 times in a row, to test how much space we have to work with in the pattern previews.